ExcelMapper/ExcelMapper.cs

```diff
@@ -466,6 +466,10 @@ object GetCellValue(ICell cell, ColumnInfo targetColumn)
         {
             var cellType = cell.CellType == CellType.Formula && targetColumn.PropertyType != typeof(string) ? cell.CachedFormulaResultType : cell.CellType;
 
+            var asString = (StringAttribute)targetColumn.Property.GetCustomAttribute(typeof(StringAttribute), false);
```

Comment:
Using reflection at every call is too costly. We should cache this in `ColumnInfo` when analyzing. Also, there should be a way to set this from code instead of attributes.

Reply:
Thank you for your quick response. I was in no way optimistic about the approach I had taken.